changes/6597.added

```diff
@@ -0,0 +1 @@
+Added an about page that displays Nautobot and support contract information.
```

Comment:
Do we want to document configuring the `NTC_SUPPORT_CONTRACT_EXPIRATION_DATE` as an "Admin action" in the 2.4 release-note?